_config.php

```diff
@@ -1,3 +0,0 @@
-<?php
-
-Object::add_extension("ModelAdmin", "VersionedModelAdmin");
```

Comment:
We have 3 options:
1. fix all this stuff and submit a pull request upstream so the author fixes it
2. fix it in our repo only, but have to deal with conflicts on upsteam updates
3. leave ours alone, with only the changes we've made.
The purist in me says (1), the pragmatist says (3)